Ginger/GingerCoreNET/Run/Context.cs

```diff
@@ -73,6 +77,7 @@ public BusinessFlow BusinessFlow
                 if (mBusinessFlow != value)
                 {
                     mBusinessFlow = value;
+                    SetAgent();
```

Comment:
remove SetAgent we should not have dependency between properties

Reply:
Removed